doc/read-the-docs-site/quick-start.rst

```diff
@@ -0,0 +1,127 @@
+.. _quick_start:
+
+Quick Start
+=================================
+
+This guide gets you started on setting up the dev environment for a simple Cardano smart contract, with the on-chain validator written in Plutus Tx.
+There are several other options for writing on-chain validators, such as `Aiken <https://aiken-lang.org/>`_ and `OpShin <https://github.com/OpShin/opshin>`_, and you can refer to their respective documentation for how to use them.
+
+If you have any questions regarding this guide, open an issue in the `Plutus issue tracker <https://github.com/input-output-hk/plutus/issues>`_.
+
+Writing the On-Chain Validator
+-----------------------------------------------------------------
+
+Prerequisites
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+GHC and Cabal should be installed in order to follow this guide.
+Plutus Tx currently supports GHC v9.2.x and v9.6.x.
+Cabal v3.8+ is recommended.
+
+Additionally, some C libraries will need to be installed.
+Plutus Tx depends on `cardano-base <https://github.com/input-output-hk/cardano-base>`_, which in turn depends on a few cryptographic C libraries, including ``libblst``, ``libsecp256k1``, and ``libsodium``.
+Cabal is not designed to manage C dependencies, so you need to either install them yourself, or manage them using another tool (such as Nix).
+
+If you are not using Nix, follow the instructions in the cardano-base repository to install these libraries (or install them in your own way if you like).
+
+If you are, you can use plutus repository's dev shell: ::
```

Comment:
Suggestion to insert "using Nix": 

"If you are using Nix, you can... "

Reply:
Done.